servatrice/scripts/db_backup_all

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
I can do the -u option, but the -e option will always fail as the script runs the ps command looking for a command result and should always return a non-zero result if there is something running.  In the case that it does return that results the error displayed will no be presented and not inform the user why the script aborts. (note: you can't see it in this revision but you will see it in the next revision of the PR shortly).